web/assets/js/util/utils.js

```diff
@@ -146,19 +143,15 @@ class RandomUtil {
 
     static randomShadowsocksPassword() {
         let array = new Uint8Array(32);
-        window.crypto.getRandomValues(array);
+        crypto.getRandomValues(array);
```

Comment:
No value added to this part. We should test if removing `window` cause any problem.

Reply:
It doesn't seem to cause any problem and browsers have no problem with it.